### DISCUSSION TITLE
Pre-release / release creates 50+ GitHub emails

### DISCUSSION THREAD

**Discussion by wiesson** [ueberdosis/tiptap#5329] [General Questions] [answered]
I'm subscribing the releases and I get like 50+ (one for each package) mails for every pre-release (and I assume a release). I'm wondering if this is a bug (I don't receive that for Svelte changes for example) or a feature. I'd like to keep notified if a new version is coming, but I don't want to have that much mails 🤓

**Comment by ruibaby** [comment]
This is probably because of the use of the [changesets](https://github.com/changesets/changesets), which seems to automatically create a Github Release for each package.

I also have this confusion, so I temporarily canceled watching release.

**Reply by nperez0111** [reply, marked as answer]
Yea sorry this is my bad, I've been releasing a lot lately for the 2.5 update.

We are discussing releasing only the packages that have changed but it is just a departure from what we have been doing for a while.
